<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMenu</class>
 <widget class="QDialog" name="AdminMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="admin_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 18px;
    font-weight: bold; 
	background-color: #222;
    color: #fff;
    border-radius: 20px;    
    padding-right: 8px;
	padding-left: 8px;
}</string>
   </property>
   <property name="text">
    <string>Admin Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manageManagersButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 20px;
    font-weight: bold; 
	background-color: #1e6091;
    color: white;
    border-radius: 20px;    
    padding: 8px; 
}

QPushButton::hover {
background-color: #1e6091;
border: 2px solid black;
}</string>
   </property>
   <property name="text">
    <string>Manage Managers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manageClientsButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 20px;
    font-weight: bold; 
	background-color: #606c38;
    color: white;
    border: 2px solid white;
    border-radius: 20px;    
    padding: 8px; 
}

QPushButton::hover {
background-color: #606c38;
border: 2px solid black;
}</string>
   </property>
   <property name="text">
    <string>Manage Clients</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manageStandsButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 20px;
    font-weight: bold; 
	background-color: #bc4749;
    color: white;
    border: 2px solid white;
    border-radius: 20px;    
    padding: 8px; 
}

QPushButton::hover {
background-color: #bc4749;
border: 2px solid black;
}</string>
   </property>
   <property name="text">
    <string>Manage Stands</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
